### PR TITLE
Update prettier config for the project.

### LINKED ISSUE #16286: Create prettier configuration for JavaScript files.

### PR DESCRIPTION
**Pull request by rahulpsingh**
fixes #16286 

### Summary

 - Update prettier config for the project.

### Review notes
 - As the configuration is very much based on my own configuration, please feel free to suggest anything you would love to see in the project. I will update the same as well.

### Testing notes
 - The application should not have any side effects.

#### Platforms

- Android
- iOS

##### Non-functional

- developer experience.

status: ready

### CODE REVIEW

**Review by ajayesivan — COMMENTED** [review]
I'm getting [this error](https://ci.status.im/blue/organizations/jenkins/status-mobile%2Fprs%2Ftests/detail/PR-16303/4/pipeline) when running `make lint` in local

**Review by ajayesivan — APPROVED** [review]
Prettier configs look good. Nice work!

Let's get approval from someone who has more experience with the repo before merging.

**Review by siddarthkay — APPROVED** [review]
Looks good! Thanks for taking the initiative @rahulpsingh !

**Review by ilmotta — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]

**Pull request merged**
